packages/ui-components/src/types.ts

```diff
@@ -1,2 +1,3 @@
 export * from './components';
 export * from './components/action-button/action-button.types';
+export * from './components/switch-button/switch-button.types';
```

Comment:
I think we need also export this interface in react application `src/components/index`. 
Also declare the component inside componeent-library.module.ts (angular lib) to export the component.

If you want to enable two way binding in `state` prop for angular, you can use `angularValueAccessorBindings` configuration inside stencil.config.js